README.md

```diff
@@ -35,7 +35,11 @@ docker run -p 3000:3000 iceman-backend
 <table>
 <tr><th>HTTP VERB</th><th>ENDPOINTS</th><th>DESCRIPTION</th></tr>
 <tr><td>POST</td><td>/api/v1/auth/signup</td><td>Creates user account</td></tr>
-<tr><td>POST</td><td>/api/v1/auth/signup</td><td>Login user account</td></tr>
+<tr><td>POST</td><td>/api/v1/auth/login</td><td>Logs user in</td></tr>
+<tr><td>GET</td><td>/api/v1/users/me</td><td>Gets user profile information</td></tr>
```

Comment:
Why 'me' in your route?

Reply:
Sorry, I forgot to change that to `/profile`. I'll do that asap.